packages/react-aria-components/src/FileTrigger.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps} from '@react-types/shared';
+import {ContextValue, DOMProps, SlotProps, useContextProps} from './utils';
+import {filterDOMProps} from '@react-aria/utils';
+import {Input} from './Input';
+import {PressResponder} from '@react-aria/interactions';
+import React, {createContext, ForwardedRef, forwardRef, useRef} from 'react';
+
+export interface FileTriggerProps extends SlotProps, DOMProps, AriaLabelingProps {
+  /**
+   * Specifies what file types are allowed.
+   */
+  accept?: string,
+  /**
+   * Whether multiple files can be selected.
+   */
+  allowsMultiple?: boolean,
```

Comment:
ah I see, the previous behavior I noticed is because the prop is `allowsMultiple` instead of `multiple`. Do we wanna just make this `multiple` to bring it in line with `capture` and `accept`? Would be nice to link to the mdn docs as well just like we do for other input props in `TextInputDOMProps`

Reply:
I'm ok with that, for the sake of matching the native API. Maybe `allowsMultiple` could be the RSP prop if we need it to be though.

Reply:
i think we discussed in the last otter time that we wanted to have the `multiple` prop have the same naming convention as other boolean props in rsp/rac, hence the allowsMultiple